Containerfile

```diff
@@ -23,7 +23,7 @@ RUN microdnf -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install \
     rpm-build \
     gcc gcc-c++ make cmake \
     cyrus-sasl-devel openssl-devel libuuid-devel \
-    python39-devel swig \
+    python39-devel python39-pip \
```

Comment:
This will work only with Proton main. All versions released up to now need swig. Should we remove swig here now, or do I need to wait for something?

Reply:
I'd leave swig until 0.39 is released, should still need pip though and also wheel I think if you want to avoid compiling the library in the run container

Reply:
done